services/movie.py

```diff
@@ -0,0 +1,42 @@
+from django.db.models import QuerySet
+
+from db.models import Movie
+
+
+def get_movies(genres_ids: list = None, actors_ids: list = None) -> QuerySet:

```

Comment:
Change your type annotation
```suggestion
def get_movies(genres_ids: list = None, actors_ids: list = None) -> QuerySet[Movie]:

```